src/Hl7.Fhir.Base/ElementModel/TypedElementParseExtensions.cs

```diff
@@ -44,13 +44,13 @@ public static class TypedElementParseExtensions
         /// <inheritdoc cref="ParseBindable"/>
         [Obsolete("WARNING! Intended for internal API usage exclusively, interface IBaseElementNavigator can be changed in " +
             "the near future.")]
-        public static Element? ParseBindableInternal<T>(this IBaseElementNavigator<T> instance) where T : IBaseElementNavigator<T>
+        public static Element? ParseBindableInternal(this ITypedElement instance) 
```

Comment:
Can we make this internal again? I remember this was just necessary for IBaseElementNavigator, but not really necessary.